lib/carrierwave/base64/adapter.rb

```diff
@@ -4,15 +4,41 @@ module Adapter
       def mount_base64_uploader(attribute, uploader_class, options = {})
         mount_uploader attribute, uploader_class, options
 
+        if options[:file_name].blank? || options[:file_name] == true
```

Comment:
1. this should be optional, because it might conflict with existing model methods
2. this should be in a separate function